src/components/subject-homepage/subject-sidebar.tsx

```diff
@@ -21,49 +21,65 @@ const SubjectSidebar = (props: Props) => {
   const [isCollapsed, setIsCollapsed] = useState(false);
 
   return (
-    <div
-      className={cn(
-        "sticky top-0 hidden max-h-screen shrink-0 bg-primary-foreground px-3 pb-12 pt-[11.5rem] lg:block",
-        "overflow-y-auto overflow-x-hidden",
-        "transition-all duration-300 ease-in-out",
-        "scrollbar scrollbar-track-primary/10 scrollbar-thumb-primary/30 scrollbar-thumb-rounded-full scrollbar-w-1.5",
-        isCollapsed ? "w-16" : "w-72",
-      )}
-    >
-      <div className="flex items-center justify-between">
-        <h2
+    <div className="z-10">
```

Comment:
Let's add `hidden lg:block` back to this to hide the sidebar on mobile screens. There's just too little space and the sidebar content is already actually included in the main section.